site/_layouts/documentation.html

```diff
@@ -68,6 +68,10 @@ <h3>Installing and Using Bazel</h3>
                 </ul>
               </li>
 
+              <li class="sidebar-nav">
+                <a href="/versions/{{ current_version }}/updating-bazel.html">Updating Bazel</a>
```

Comment:
Why would this be versioned? The current process is the only one which should be used.

Reply:
It is versioned because everything else is versioned (the future master might not have `updating-bazel.md` and the link will break)